### PR TITLE
Modifications on oscSender

### PR DESCRIPTION
**Pull request by frankiezafe**
- OSCtransmitter now inherits Node and not Spatial (seems more relevant)
- oscSender.init requires to define the buffer size - i didn't found a way to create a init(host,port) and a init(host,port,buffersize) that compiles with gdscript, so the only one accessible now is init(host,port,buffersize)
- script attached to an OSCtransmitter uses the methods of the instance directly (see readme)

i'm waiting for a reply on how to create an icon for the object

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,33 +108,35 @@ func _on_osc_message(val):
...
 ```
 
-And here's an approach for sending:
+### Sending messages
+
+First of all, you need to add an **OSCtransmitter** node in your scene. It is located in the first level of the tree, between *HTTPRequest* and *ResourcePreloader*.
+
+![OSCtransmitter in creation menu](https://frankiezafe.org/images/7/7c/Godot_gdosc_OSCtransmitter.png)
+
+Once done, attach a script to it who looks like this:
```
Comment (djiamnot) [inline README.md]: ... attach a script _that_ looks ...

Comment (frankiezafe) [reply README.md]: you begin to read my text :)
modification will be part of the next pull request, i will cancel this one

Comment (djiamnot) [reply README.md]: Yeah I figured, I might as well, since you're getting serious...

Comment (frankiezafe) [reply README.md]: :) - i know it's super nerdy, but i'm glad the object has an icon in the editor -> https://frankiezafe.org/images/a/ab/Godot_osctransmitter_icon.png
I need to find a way to add it without "hacking" the editor... I'm waiting for a response from the community.

**Pull request merged**